src/create-element.js

```diff
@@ -95,6 +95,7 @@ export function coerceToVNode(possibleVNode) {
 	if (possibleVNode._dom!=null) {
 		let vnode = createVNode(possibleVNode.type, possibleVNode.props, possibleVNode.text, possibleVNode.key, null);
 		vnode._dom = possibleVNode._dom;
+		vnode._original = possibleVNode._original || possibleVNode;
```

Comment:
This had to be added since we now possibly coerce two times, once when newChildren get added and once before diffing.

Reply:
We could add another argument for `original` to `createVNode`. That would also ensure the object's shape is unchanged, which is needed for performance.